example/jupyter/tutorial_analyzer.ipynb

```diff
@@ -0,0 +1,230 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Analyzer](../../doc/analyzer_design.md) is designed to explain the machine learning model in SQLFlow. Here we are going to learn how to write an `ANALYZE` SQL.  \n",
```

Comment:
Missing documentation titles. Maybe the following paragraphs would help.

# Analyzing model on SQLFlow Tutorial

In this tutorial, we will learn to
- Train an XgBoost tree model on [the Boston housing dataset](https://www.kaggle.com/c/boston-housing).
- Analyze the trained model using `ANALYZE` SQL statements.

You can find more SQLFlow usage from the [User Guide](https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/user_guide.md).